Core/Main/itkElastixRegistrationMethod.hxx

```diff
@@ -825,7 +825,7 @@ ElastixRegistrationMethod<TFixedImage, TMovingImage>::GetNumberOfTransforms() co
 template <typename TFixedImage, typename TMovingImage>
 auto
 ElastixRegistrationMethod<TFixedImage, TMovingImage>::GetNthTransform(const unsigned int n) const
-  -> SmartPointer<TransformType>
+  -> TransformType *
```

Comment:
OK, you convinced me here: it does indeed look fine to me to replace the smart pointer by a raw pointer here, because the "NthTransform" is still owned (and kept alive) by ElastixRegistrationMethod. Thanks Matt.